docker-compose build ERROR: unsatisfiable constraints: required by: world[python]

FROM alpine:latest 
需要修改为
FROM alpine:3.11

if I change line 7 in the `opentsdb.Dockerfile` file.
From this
```
ARG BUILD_PACKAGES="build-base autoconf make automake git python cairo-dev pango-dev gd-dev lua-dev readline-dev libpng-dev libjpeg-turbo-dev libwebp-dev"
```
To this _(note I am using `python3` as a dependency)_
```
ARG BUILD_PACKAGES="build-base autoconf make automake git python3 cairo-dev pango-dev gd-dev lua-dev readline-dev libpng-dev libjpeg-turbo-dev libwebp-dev"
```
That error is no longer present. Instead, I get:

```
set dummy "https://repo1.maven.org/maven2/org/apache/zookeeper/zookeeper/3.4.6" "third_party/zookeeper/zookeeper-3.4.6.jar"; shift; ./build-aux/fetchdep.sh "$@"
Connecting to repo1.maven.org (199.232.196.209:443)
saving to 'third_party/zookeeper/zookeeper-3.4.6.jar-t'
zookeeper-3.4.6.jar- 100% |********************************|  774k  0:00:00 ETA
'third_party/zookeeper/zookeeper-3.4.6.jar-t' saved
../build-aux/gen_build_data.sh src/tools/BuildData.java net.opentsdb.tools 2.4.0
Generating src/tools/BuildData.java
../build-aux/gen_build_data.sh: line 36: python: command not found
make: *** [Makefile:1842: src/tools/BuildData.java] Error 127
ERROR: Service 'opentsdb' failed to build : The command '/bin/sh -c cd /tmp     && curl -OL https://github.com/OpenTSDB/opentsdb/archive/v2.4.0.zip     && unzip v2.4.0.zip     && mv opentsdb-2.4.0 ${TSDB_DIR}     && rm /tmp/v2.4.0.zip     && cd ${TSDB_DIR}     && find . -name '*.mk' | xargs sed -i s#http://central.maven.org#https://repo1.maven.org#g     && find . -name '*.mk' | xargs sed -i s#http://repo1.maven.org#https://repo1.maven.org#g     && ./build.sh' returned a non-zero code: 2
```

--- a/docker/opentsdb.Dockerfile
+++ b/docker/opentsdb.Dockerfile
@@ -4,7 +4,7 @@ ARG HBASE_VERSION=2.2.4
 ARG GNUPLOT_VERSION=5.2.8
 
 ARG PACKAGES="ca-certificates rsyslog bash openjdk8 curl libgd libpng libjpeg libwebp libjpeg-turbo cairo pango jruby lua supervisor asciidoctor"
-ARG BUILD_PACKAGES="build-base autoconf make automake git python cairo-dev pango-dev gd-dev lua-dev readline-dev libpng-dev libjpeg-turbo-dev libwebp-dev"
+ARG BUILD_PACKAGES="build-base autoconf make automake git python3 cairo-dev pango-dev gd-dev lua-dev readline-dev libpng-dev libjpeg-turbo-dev libwebp-dev"
 
 ARG DOCKER_ROOT="docker"
 ARG HBASE_DIR=/hbase
@@ -18,7 +18,8 @@ ENV PATH="$JAVA_HOME/bin:${PATH}"
 
 # Install dependencies
 RUN apk --update add apk-tools \
-    && apk add ${PACKAGES} ${BUILD_PACKAGES}
+    && apk add ${PACKAGES} ${BUILD_PACKAGES} \
+    && ln -sf python3 /usr/bin/python
 
 WORKDIR /tmp/gnuplot
 RUN cd /tmp \

